src/urbit/mar/chess/moves.hoon

```diff
@@ -0,0 +1,30 @@
+::  take in a chess-game, assemble all moves into a list.
```

Comment:
Looks good. Good solution, given out ugly, typeless moves array. Could you add a to-do comment to line `219` in `sur/chess.hoon` that we should probably have a named type for `[chess-move chess-fen chess-san]` (and possibly a list thereof)?